css/admin.css

```diff
@@ -139,6 +139,14 @@ ul#kaltura-browse li .showName {
 	margin:      0 5px 0 5px;
 }
 
+ul#kaltura-browse li .entryTitle {
```

Comment:
There is duplication in CSS here, maybe use `.entryTitle` (or change it to something else) as the base class (that will always be added to the div) and `.showName` to initialize `kalturaEditableName` plugin.